src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.tHead.querySelectorAll('th');
+
+headers.forEach(header => {
+  header.addEventListener('click', () => {
+    sortTable(header);
+  });
+});
+
+function sortTable(column) {
+  const rows = Array.from(table.tBodies[0].querySelectorAll('tr'));
+
+  rows.sort((a, b) => {
+    const aValue = a.cells[column.cellIndex].textContent;
+    const bValue = b.cells[column.cellIndex].textContent;
+    const aNumber = convertToNumber(aValue);
+    const bNumber = convertToNumber(bValue);
+
+    if (!aNumber) {
```

Comment:
Instead of checking `!aNumber`, it's better to use `isNaN(aNumber)` for checking if the value is not a number.